static/app/components/datePageFilter.tsx

```diff
@@ -77,6 +100,7 @@ function DatePageFilter({router, resetParamsOnChange, ...props}: Props) {
         onUpdate={handleUpdate}
         label={<IconCalendar color="textColor" />}
         dateSummary={getDateSummary}
```

Comment:
Now that we have custom dropdown functionality I think we can remove this `dateSummary` and the `dateSummary` prop as a whole in the `timeRangeSelector` component since it was only used in the case of this page filter.

Reply:
oh good catch! removed in https://github.com/getsentry/sentry/pull/32005/commits/3a540ffabb54916ec1a94b847ff59cbc3556ce72